### DISCUSSION TITLE
[Script request] RustDesk - open source alternative to TeamViewer

### DISCUSSION THREAD

**Discussion by tOOmki** [community-scripts/ProxmoxVE#112] [Request script]
### Application Name

RustDesk

### Website

https://rustdesk.com/docs/en/self-host/

### Description

RustDesk - open source alternative to TeamViewer.
Link to previous request:
https://github.com/tteck/Proxmox/discussions/3691

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by florianwilde** [comment]
This is already open Script request.
#34 
Please close

**Comment by Mellowlynx** [comment]
Indeed, duplicate of #34
